push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java

```diff
@@ -134,32 +134,37 @@ public void loadAndQueueTokenBatch(@Observes @Dequeue MessageHolderWithVariants
                 String lastTokenInBatch = null;
                 int tokensLoaded = 0;
                 for (int batchNumber = 0; batchNumber < configuration.batchesToLoad(); batchNumber++) {
+
+                    // increasing the serial ID,
+                    // to make sure it's properly read from all block
+                    ++serialId;
```

Comment:
yes, it's an increment, based on the batch.

Currently the increase only happens here:
* https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java#L144

With the consequence that in other blocks the value is different, like: 
* https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java#L147
* or here, were we added the new log -> https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java#L159